spec/models/food_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Food, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
- [ ] Your implementations are great, But please remember we are expected to write Unit and Integration tests! as part of this milestone. As professional developers, we need to be able to write tests to make sure everything states in place.

Reply:
Hi @TadesseDev, Since they're not a core requirement, I left them for later implementation.

Reply:
Dear @DaveZag, Actually they are core requirements. This is among the things we are looking for in the project. sorry brother but I'm sure you can manage 🙌